Add author component

hirundinidae opened the pull request:
Adds author component to use on blog post card and individual blog pages. 
- not sure if the width works for the card/use on pages because I haven't created the blog card, but that might need adjusting 
- joining multiple authors together is separated by `, ` but since most blog posts have at most 2 authors, this could be changed to ` and `

![screenshot_20210812_000142](https://user-images.githubusercontent.com/65516245/129136535-c1a0dc6d-f5f0-414c-8a85-fc92d36a1a3c.png)
![screenshot_20210812_000137](https://user-images.githubusercontent.com/65516245/129136536-92140349-57d9-4d04-98b7-5d9a30c1d153.png)

hirundinidae committed:
Add author components

hirundinidae committed:
Format files

hirundinidae committed:
Add authors as props and fix margins

hirundinidae committed:
Add proptypes and comments

xsharonhe commented on components/Author.jsx:
```diff
@@ -0,0 +1,56 @@
+      <Image src={avatar} width={50} height={50} />
+    </Avatar>
+    <Names>
+      {names.length == 1 ? (
```
Should be === because JavaScript equalities

xsharonhe reviewed (commented):
Kinda my fault but I fully intended that if there is more than one person that the default profile picture is a tie-dye background (the same one as impact report background), but we can get this merged in first and you can change it for the next one after fixing merge conflicts lol

hirundinidae committed:
Make suggested changes

set default profile picture to tie dye background
delete previous profile pictures svg
change to ===

xsharonhe commented:
You need to fix merge conflicts or else it won't merge :0

hirundinidae committed:
Merge branch 'main' of github.com:PuMProfessionals/pump-web-v2 into add-author

hirundinidae committed:
Format files

xsharonhe reviewed (approved):
I will probably change the logic about DefaultProfile of this a bit later but it's good to merge and LGTM
